src/gossip_agent/message_typing.rs

```diff
@@ -53,11 +54,13 @@ fn prepare_nonces(
 }
 
 pub async fn get_indexer_stake(address: String) -> Result<BigUint, anyhow::Error> {
-    Ok(
-        query_network_subgraph(NETWORK_SUBGRAPH.to_string(), address.clone())
-            .await?
-            .indexer_stake(),
-    )
+    dotenv().ok();
+
+    let network_subgraph = env::var("NETWORK_SUBGRAPH").expect("No NETWORK_SUBGRAPH set.");
```

Comment:
maybe let's process the environmental variables upon radio startup and initialize gossip agent with them? This way setup warnings is shown earlier on, and this particular function doesn't need to access env everytime

Reply:
that's a good idea, or maybe if none are provided we can set these as default ones on the SDK side

Reply:
Sure, I can add them as fields on `GossipAgent`, although it feels like there are a lot of setup fields already, maybe post-MVP we can use a yaml config file for this (we can give the users the option to construct with fields using `new` or load a yaml config with `from_file` or something), I'll add an Issue for that

Reply:
I like it because it's good to keep env vars outside the SDK

Reply:
Done ✅